### ISSUE TITLE
links to detail tabs not working any more

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#219]
The links to detail tabs are not working any more:

![Screenshot_20200701-110706(1)](https://user-images.githubusercontent.com/323719/86226335-cd73fb80-bb8b-11ea-86c8-b785ac6476ca.jpg)

And in general, links are (e.g. to switch between tabs in the intro) are less responsive than before.

**Comment by opeeters** [comment]
Console:
```
ERROR Error: "Parameter "key" required"
```
endless loop until script is stopped.

**Comment by opeeters** [comment]
I rolled back to previous commits.. can't find a working state.

**Comment by janschulte** [comment]
I currently check it. There are strange response for the belaqi sometimes: 

Currently: https://geo.irceline.be/forecast/wms?service=WMS&request=GetFeatureInfo&version=1.1.1&layers=forecast:belaqi_d3&info_format=application/json&width=1&height=1&srs=EPSG:4326&bbox=4.1180489,50.795271099999994,4.118058899999999,50.7952811&query_layers=forecast:belaqi_d3&X=1&Y=1&time=2020-07-01T06:10:39.000Z

Reponses with: 
```
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "id": "",
      "geometry": null,
      "properties": {
        "GRAY_INDEX": -9999
      }
    }
  ]
}
```

I will fix wrong handling of this responses in the code.

**Comment by opeeters** [comment]
`-9999` is the no data value. The app should be able to handle this. In a real-time dataflow everything can and will go wrong at some stage..

**Comment by opeeters** [comment]
This will explain this problem as well:

![Screenshot_20200701-111214](https://user-images.githubusercontent.com/323719/86230928-0dd67800-bb92-11ea-80f8-e6a61643654f.jpg)

This would be de `d3` position in the timeline. If there is a `-9999` due to some issue in the forecasting chain, for `d2` and `d1` you could check for cached data, but in the case of `d3`, just hide the element..

**Comment by janschulte** [comment]
Fixed with: 9dc29e2a503349a1060007ae8c47b6cfd7a9be5d

**Comment by opeeters** [comment]
Thanks for the fast fix!
